### PR TITLE
Update known-issues.md

### PR DESCRIPTION
**Pull request by andrewnyr**
channel name change @signalnerve :)

### CODE REVIEW

**Review by kodster28 — APPROVED** [review]
Approving from a content perspective (though I don't know the actual name of the channel).

**Review by kodster28 — COMMENTED** [review]
@deadlypants1973, could you confirm the name of the repo and — if it's correct — merge in?

**Comment by deadlypants1973** [comment]
> @deadlypants1973, could you confirm the name of the repo and — if it's correct — merge in?

that is correct.

File: products/pages/src/content/platform/known-issues.md [hunk 1]
```diff
@@ -8,4 +8,4 @@ Here are some known bugs and issues that we're aware of with Cloudflare Pages:
 - Hugo builds automatically run an old version. To run the latest version of Hugo (for instance, 0.80.0), you'll need to set an environment variable. Set `HUGO_VERSION` to `0.80.0` or the Hugo version of your choice.
 - By default, we use Node 10 in the Pages build environment. If you need to use a newer Node version, see our [Build configuration page](https://developers.cloudflare.com/pages/platform/build-configuration) for configuration options.
 
-Having an issue that you don't see listed above? Let us know in the Cloudflare Workers Discord! Get your invite at [discord.gg/cloudflaredev](https://discord.gg/cloudflaredev), and share your bug report in the #pages channel.
+Having an issue that you don't see listed above? Let us know in the Cloudflare Workers Discord! Get your invite at [discord.gg/cloudflaredev](https://discord.gg/cloudflaredev), and share your bug report in the #pages-help channel.
```
Comment (deadlypants1973) [inline products/pages/src/content/platform/known-issues.md:11]:
```suggestion
Having an issue that you don't see listed above? Let us know in the Cloudflare Workers Discord. Get your invite at [discord.gg/cloudflaredev](https://discord.gg/cloudflaredev), and share your bug report in the #pages-help channel.
```

**Review by deadlypants1973 — APPROVED** [review]
Changed punctuation on line 11, @signalnerve or @lukeed to merge.

**Pull request merged**